src/Config.js

```diff
@@ -37,11 +37,12 @@ export class Config {
 
     // Create a new DatabaseController per request
     if (cacheInfo.databaseController) {
-      const schemaCache = new SchemaCache(cacheInfo.cacheController, cacheInfo.schemaCacheTTL);
+      const schemaCache = new SchemaCache(cacheInfo.cacheController, cacheInfo.schemaCacheTTL, cacheInfo.freezeSchema);
```

Comment:
each schemaCache instance has special keys prefix, so i think per request  would not share the SchemaCache  
```
  this.prefix = SCHEMA_CACHE_PREFIX+randomString(20);
```